app/tweep_dm.py

```diff
@@ -0,0 +1,223 @@
+list_of_welcome_messages = [
+    {
+        'welcome_message': {
+            'id': '1424836133582774286',
+            'created_timestamp': '1628542381078',
+            'message_data': {
+                'text': 'new message',
+                'entities': {
+                    'hashtags': [],
+                    'symbols': [],
+                    'user_mentions': [],
+                    'urls': []
+                }
+            },
+            'source_app_id': '21602950',
+            'name': 'new name'
+        },
+        'apps': {
+            '21602950': {
+                'id': '21602950',
+                'name': 'bluewitness1'
+            }
+        }
+    },
+    {
+        'welcome_message': {
+            "id": "1424835217869709320",
+            "created_timestamp": "1628542162755",
+            "message_data": {
+                "text": "new message",
+                "entities": {
+                    "hashtags": [],
+                    "symbols": [],
+                    "user_mentions": [],
+                    "urls": []
+                },
+                "quick_reply": {
+                    "type": "options",
+                    "options": [
+                        {
+                            "label": "Rank 1",
+                            "metadata": "force_rank_1",
+                            "description": "Non-violent Police Presence."
+                        },
+                        {
+                            "label": "Rank 2",
+                            "metadata": "force_rank_2",
+                            "description": "Open Handed (Arm Holds & Pushing)"
+                        },
+                        {
+                            "label": "Rank 3",
+                            "metadata": "force_rank_3",
+                            "description": "Blunt Force Trauma (Batons & Shields)"
+                        },
+                        {
+                            "label": "Rank 4",
+                            "metadata": "force_rank_4",
+                            "description": "Chemical & Electric Weapons (Tasers & Pepper Spray)"
+                        },
+                        {
+                            "label": "Rank 5",
+                            "metadata": "force_rank_5",
+                            "description": "Lethal Force (Guns & Explosives)"
+                        }
+                    ]
+                }
+            }
+        }
+    }
+]
+# Change to dict with keys of date location, and confirmation. Values list of A/B text
+list_of_A_B_txts = [
+    'Hi! I am a bot for Blue Witness, a project by @humanrights1st. We noticed your tweet may involve police misconduct, please confirm the date of this incident here: ',
+    'Hi! I am a bot for Blue Witness, a project by @humanrights1st. We noticed your tweet may involve police misconduct, please confirm the location of this incident here: ',
```

Comment:
I love what you did here with this introduction! Gives the user proper context for why our bot is reaching out to them.

Reply:
These introductions are great! It explains all relevant information needed by the user about this project and asks for specification on either date or location.  Great job on these!